src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
+const MAX_HEALTHS = 100;
```

Comment:
The constant `MAX_HEALTHS` should be singular, `MAX_HEALTH`, since it represents a single value.

Reply:
> The constant `MAX_HEALTHS` should be singular, `MAX_HEALTH`, since it represents a single value.

Good point, and probably it's DEFAULT_HEALTH, not max